ffmpeg/videoprofile.go

```diff
@@ -36,6 +36,25 @@ const (
 	GOPInvalid = -2
 )
 
+// for sceneclassification
+const (
```

Comment:
```suggestion
type DNNFilterType int

const (
    DNNFilterType DNNClassify = iota
    ...
```

Since there is a concept of "filter type" in https://github.com/livepeer/FFmpeg/pull/1 we could use a custom enum type to represent that.

Reply:
Separate question - should the values for this enum be strings since the filter type described in the ffmpeg PR are strings i.e. "lvpclassify" and "lvpodetect"